src/components/grid-aware/DonationBlock/DonationBlock.module.css

```diff
@@ -74,6 +74,18 @@
   margin-top: 5px;
 }
 
+.mainTitleImage {
+  height: auto;
+  margin-bottom: 0;
+  margin-top: 5px;
+  max-width: 100%;
+}
+
+.givingTuesdayParagraph {
+  font: var(--font-body-medium);
+  margin-top: 40px;
```

Comment:
This needs to be decreased in mobile because 40px is far too large.

Reply:
ok